core/domain/exp_domain.py

```diff
@@ -185,6 +188,11 @@ class ExplorationChange(change_domain.BaseChange):
             'state_name', 'content_id', 'language_code', 'content_html',
             'translation_html'],
         'optional_attribute_names': []
+    }, {
+        'name': CMD_ADD_CUSTOMIZATION_ARG,
```

Comment:
Is this action even possible in the UI? If not, then we should not add this.

Reply:
The reason for adding this was to be able to check the type of interaction, but since it's not needed anymore anyways, I'm removing this.